v2/README.md

```diff
@@ -60,9 +53,7 @@ type CallOptions func(i interface{})
 type Store interface {
 	NewTransaction(ctx context.Context, records ...Record) (StoreTransaction, error)
 	LoadByAggregate(ctx context.Context, aggregateID string, opts ...QueryOptions) (record []Record, err error)
-	LoadBySequenceID(ctx context.Context, sequenceID string, opts ...QueryOptions) (record []Record, err error)
```

Comment:
bryter inte det här legacy?

Reply:
From what I can see the Iot and measurement service uses github.com/SKF/go-eventsource/v2, maybe should v3 be introduced?

Reply:
That kinda the point with v2, to introduce and use queryoptions.
But yeah, in practice it breaks the v2.0.0 and v2.1.0, would love to keep it in the v2

Reply:
It will affect two service managed by system, i can take responsibility to bump those services (and all users that uses this outside of SKF)